medicinski-razgledi.csl

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" delimiter-precedes-et-al="always" delimiter-precedes-last="always" initialize-with="" demote-non-dropping-particle="sort-only" default-locale="sl-SI">
+  <info>
+    <title>Medicinski razgledi</title>
+    <title-short>Med. razgl.</title-short>
+    <id>http://www.zotero.org/styles/medicinski-razgledi</id>
+    <link href="http://www.zotero.org/styles/medicinski-razgledi" rel="self"/>
+    <link href="http://www.zotero.org/styles/revista-da-sociedade-brasileira-de-medicina-tropical" rel="template"/>
+    <link href="https://medrazgl.si/navodila-sodelavcem/" rel="documentation"/>
+    <author>
+      <name>Luka Petravić</name>
+      <email>lpetravic@me.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>0025-8121</issn>
+    <eissn>1581-3797</eissn>
+    <updated>2021-12-06T19:54:16+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author" font-weight="normal" delimiter=",">
+      <name delimiter-precedes-et-al="always" delimiter-precedes-last="always" et-al-min="3" initialize-with="" name-as-sort-order="all" sort-separator="  "/>
```

Comment:
You have an et-al-min here without an et-al-use-first, which isn't technically invalid, but is confusing in styles (and may have poorly determined output across different processors) so we're requiring both elements for the repository

Reply:
@adam3smith  Thank you for your prompt reply. How should i go about adressing this issue to resolve it?

Reply:
Just adding et-al-use-first="1" (which I assume is what you want?) will fix the test

Reply:
<img width="948" alt="image" src="https://user-images.githubusercontent.com/29688990/146523661-674f568b-16c7-4e89-8e3f-b5ea17492b6f.png">

I'm sorry @adam3smith but the problem persists... Anything else I could do?

Reply:
Same issue in line 85

Reply:
Works now, thank you for your time and knowledge! @adam3smith